src/iop/basecurve.c

```diff
@@ -966,7 +966,7 @@ static inline void apply_legacy_curve(
       else
         out[k+i] = dt_iop_eval_exp(unbounded_coeffs, f);
     }
-    out[k+3] = in[k+3];
+    out[k+3] = fmaxf(in[k+3], 0.f);
```

Comment:
I don't see how in[] can be negative or if it is then the input data in basecurve has created them as in is a pointer to ivoid passed to process() routine. My thinking is that it would be better to fmax() the output of whatever modules before basecure has created the negative values, right?

Reply:
Yeah, this is the alpha channel anyway. I will change that tomorrow.